lib/cute_pets.rb

```diff
@@ -6,16 +6,43 @@
 module CutePets
   extend self
 
-  def post_pet()
-    pet = nil
+  def fetch_pet()
     if ENV.fetch('pet_datasource').downcase == 'petfinder'
       pet = PetFetcher.get_petfinder_pet
     else
       pet = PetFetcher.get_petharbor_pet
     end
+  end
+
+  def post_pet()
```

Comment:
Assignment Branch Condition size for post_pet is too high. [24.49/15]<br>Omit the parentheses in defs when the method doesn't accept any arguments.